scipy/integrate/_ivp/ivp.py

```diff
@@ -398,6 +398,14 @@ def solve_ivp(fun, t_span, y0, method='RK45', t_eval=None, dense_output=False,
       0.05858998 0.02701876]
      [8.         7.5534414  4.25308709 1.73277247 0.7059579  0.287618
       0.11717996 0.05403753]]
+    
+    >>> import matplotlib.pyplot as plt
+    >>> plt.plot(sol.t, sol.y[0], label='y0 = 2')
+    >>> plt.plot(sol.t, sol.y[1], label='y0 = 4')
+    >>> plt.plot(sol.t, sol.y[2], label='y0 = 8')
+    >>> plt.legend()
+    >>> plt.xlabel('$t$')
```

Comment:
I suggest adding ylabel equal to '$y$'.

Reply:
Even though the property is not specified? If the example would explicitly refer to a concentration or any physical quantity I would have put it there in the first place. So I was unsure.

In the end it's up to you.

Reply:
As `y` is used everywhere in the documentation of `solve_ivp` as the unknown function it makes sense to put this label.